loops.js

```diff
@@ -23,13 +23,16 @@ for (let i = 0; i < 3; i++) {
  // ==========================================
 
 for (let i = 0; i < 5; i++) {
-    let words = "Loop";
-    console.log(words);
-
-    if (let words[i] = "klaar!"{
-        console.log(words);
+    if (i < 4) {
+        console.log("loop")
```

Comment:
Vergeet niet om consequent aan het einde een ; of géén ; te gebruiken maar niet door elkaar.